### DISCUSSION TITLE
Add table name to top of UI

### DISCUSSION THREAD

**Discussion by rphly** [supabase/supabase#19097] [Feature Requests]
I know this is kinda frivolous cause the table name is on the left of the table editor, but i swear it would be a huge QOL upgrade to just have the name on top (because of the horizontal space for longer names) and also cause it's just natural to look there for the name. thanks a bunch team 

![Screenshot 2023-11-20 at 5 46 12 PM](https://i.imgur.com/dlWUjbX.png)

**Comment by Hallidayo** [comment]
Hi @rphly - I've just created a [pull](https://github.com/supabase/supabase/pull/19152) for this feature request

**Comment by saltcod** [comment]
Hey all — thanks for the request and the PR. I'm going to close this (and the PR) in the name of UI simplification. Without near constant vigilance, the ui will be made up of a thousand labels and buttons. Appreciate the request (and the PR!)
